### ISSUE TITLE
Contact form ignores theme

### ISSUE THREAD

**Issue by DarrenCook** [greglobinski/gatsby-starter-personal-blog#23]
If you bring in the dark background from new-theme, the contact form questions are black on black. (Each input is green on black when given focus.)

I haven't yet worked out how to fix this. E.g. I tried adding this in the `singleLineInput` styles block:

    color: theme.main.colors.text,

but it makes no difference.

**Comment by greglobinski** [comment]
Thank you @DarrenCook 

First, there is a bug in `theme.js` (in `new-theme.js` also)
```
  palette: {
    primary: {
      main: colors.accent
    }
  },
  typography: {
    fontFamily: `Arial, sans-serif`,
    fontSize: 16
  },
  pallete: {
    action: {
      hover: "rgba(0, 0, 0, 0.01)"
    }
  }
```
the `pallete` property is a misspelled `palette` so it should be removed and its inner properties have to be moved to the `palette`, like below

```
  palette: {
    primary: {
      main: colors.accent
    },
    action: {
      hover: "rgba(0, 0, 0, 0.01)"
    }
  },
  typography: {
    fontFamily: `Arial, sans-serif`,
    fontSize: 16
  }
```
and now to fix the form's colors for the dark `new-theme.js` we have to add a new inner property to `palette` `type: "dark"`, the object should look like below.

```
  palette: {
    primary: {
      main: "#709425"
    },
    type: "dark",
    action: {
      hover: "rgba(0, 0, 0, 0.01)"
    }
  },
```
with that the form will look as it should, more about that in Material UI docs https://material-ui-next.com/customization/themes/

I will update the starter's code and the article (https://forgatsby.greglobinski.com/customize-personal-blog-starter/) within a few days.

**Comment by doakheggeness** [comment]
These changes to the ``` new-theme.js ``` &/or ``` theme.js ``` make the contact form inputs hidden.

**Comment by greglobinski** [comment]
@doakheggeness 
What do you mean by "make the contact form inputs hidden"?

**Comment by doakheggeness** [comment]
The form was blank. except for the 'Send' button on load. If you clicked in an area, for instance where the 'name' should be displaying, then it would act normally ( name minimizing, etc.) and be visible. It appears the style for the inputs must be #fff, matching the background?? When I changed the two stylesheets back to the previous state, everything displayed as normal. I am a newby to React, so I am slowing getting the hang of of.

**Comment by greglobinski** [comment]
@doakheggeness 
I suggest reading the Material UI v. 1.0 docs about theming.  https://material-ui-next.com/customization/themes/

**Comment by greglobinski** [comment]
Fixed with version 1.1.0
